code/game/machinery/doors/airlock.dm

```diff
@@ -1378,21 +1378,36 @@ GLOBAL_LIST_EMPTY(airlock_emissive_underlays)
 		shock(user, 100) //Mmm, fried xeno!
 		return
 	if(!density) //Already open
+		if(!arePowerSystemsOn())
+			close(TRUE)
+			return
+		if(user.queen_strength == 1)
+			close(TRUE)
+			return
 		return
 	if(locked || welded) //Extremely generic, as aliens only understand the basics of how airlocks work.
 		to_chat(user, "<span class='warning'>[src] refuses to budge!</span>")
 		return
 	if(prying_so_hard)
 		return
 	prying_so_hard = TRUE
-	user.visible_message("<span class='warning'>[user] begins prying open [src].</span>",\
-						"<span class='noticealien'>You begin digging your claws into [src] with all your might!</span>",\
-						"<span class='warning'>You hear groaning metal...</span>")
-	var/time_to_open = 5
+	var/time_to_open
 	if(arePowerSystemsOn())
+		if(user.queen_strength == 1)
+			open(TRUE)
+			visible_message("<span class='danger'>[user] forces the door!</span>")
+			playsound(loc, "sparks", 100, TRUE, SHORT_RANGE_SOUND_EXTRARANGE)
+			prying_so_hard = FALSE
+			return
+		user.visible_message("<span class='warning'>[user] begins prying open [src].</span>",\
+							"<span class='noticealien'>You begin digging your claws into [src] with all your might!</span>",\
+							"<span class='warning'>You hear groaning metal...</span>")
 		time_to_open = 50 //Powered airlocks take longer to open, and are loud.
 		playsound(src, 'sound/machines/airlock_alien_prying.ogg', 100, TRUE)
-
+	else
+		time_to_open = 0
```

Comment:
You can remove this line if you just set the `time_to_open` var to be `0` when you declare it